Library/Homebrew/extend/time.rb

```diff
@@ -1,6 +1,8 @@
 # typed: true
 # frozen_string_literal: true
 
+require "time"
```

Comment:
This is needed to alias `xmlschema` below (in contexts where it has not already been included, such as `SystemCommand` in this new layout).